### PR TITLE
fix: Show the hidden text on about-us page

### LINKED ISSUE #1825: Words in about us page are missing
**Actual Behaviour**

Some words are missing in the about us page

**Expected Behaviour**

Missing words should be displayed

**Screenshots of the issue**

screenshot_1546955453

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by 091karan**
Fixes #1825 

Changes: 
Added the code in the AboutUsFragement to change the HTML strings into displayabl text.

Screenshots for the change: 

Before

screenshot_1546955453

After

screenshot_1546955321

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/skills/aboutus/AboutUsFragment.kt [hunk 1]
```diff
@@ -43,29 +45,61 @@ class AboutUsFragment : Fragment() {
...
+            susi_license_info_desc.text = Html.fromHtml(getString(R.string.susi_license_information_desc), Html.FROM_HTML_OPTION_USE_CSS_COLORS) // for 24 api and more
+        } else {
+            susi_license_info_desc.text = Html.fromHtml(getString(R.string.susi_license_information_desc)) // or for older api
+        }
+
+        if (Build.VERSION.SDK_INT >= 24) {
+            susi_report_issues_desc.text = Html.fromHtml(getString(R.string.susi_report_issues_desc), Html.FROM_HTML_OPTION_USE_CSS_COLORS) // for 24 api and more
+        } else {
+            susi_report_issues_desc.text = Html.fromHtml(getString(R.string.susi_report_issues_desc)) // or for older api
+        }
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/skills/aboutus/AboutUsFragment.kt]: This is what we call duplication. And duplication is bad. Create a single method to handle the HTML returns

Comment (091karan) [reply app/src/main/java/org/fossasia/susi/ai/skills/aboutus/AboutUsFragment.kt]: @iamareebjamal I fixed duplication. Please review. Thanks.

File: app/src/main/java/org/fossasia/susi/ai/skills/aboutus/AboutUsFragment.kt [hunk 2]
```diff
@@ -43,29 +45,37 @@ class AboutUsFragment : Fragment() {
 
     @NonNull
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-        about_susi.setOnClickListener({
+
+        about_susi.text = Html.fromHtml(getString(R.string.susi_about))
+        contributors_desc.text = Html.fromHtml(getString(R.string.susi_contributors_desc))
+        susi_skill_cms_desc.text = Html.fromHtml(getString(R.string.susi_skill_cms_desc))
+        susi_license_info_desc.text = Html.fromHtml(getString(R.string.susi_license_information_desc))
+        susi_report_issues_desc.text = Html.fromHtml(getString(R.string.susi_report_issues_desc))
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/skills/aboutus/AboutUsFragment.kt]: This is deprecated, just create a method which checks API and returns correct HTML call

Comment (091karan) [reply app/src/main/java/org/fossasia/susi/ai/skills/aboutus/AboutUsFragment.kt]: @iamareebjamal Added method. Please review.

File: app/src/main/java/org/fossasia/susi/ai/skills/aboutus/AboutUsFragment.kt [hunk 3]
```diff
@@ -43,14 +44,23 @@ class AboutUsFragment : Fragment() {
         super.onPrepareOptionsMenu(menu)
     }
 
+    private fun htmlConverter(resourceId: Int): Spanned{
+        if(Build.VERSION.SDK_INT>24){
+            return Html.fromHtml(getString(resourceId),Html.FROM_HTML_OPTION_USE_CSS_COLORS)
+        }
+        else{
```
[the remark this answers is not in the document]
Comment (091karan) [reply app/src/main/java/org/fossasia/susi/ai/skills/aboutus/AboutUsFragment.kt]: @iamareebjamal Done. Please review.

File: app/src/main/java/org/fossasia/susi/ai/skills/aboutus/AboutUsFragment.kt [hunk 4]
```diff
@@ -43,14 +44,23 @@ class AboutUsFragment : Fragment() {
         super.onPrepareOptionsMenu(menu)
     }
 
+    private fun htmlConverter(resourceId: Int): Spanned{
```
[the remark this answers is not in the document]
Comment (091karan) [reply app/src/main/java/org/fossasia/susi/ai/skills/aboutus/AboutUsFragment.kt]: @iamareebjamal Done. Please review.

**Comment by naman653** [comment]
I think this issue has already been solved.

**Pull request closed without merging**